README.md

```diff
@@ -42,10 +41,6 @@
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/230090471-3c6d2e2e-da70-4a2a-bb4f-30f46179f556.png)

**[OPTIONAL]**

- [ ] Kindly remove the top title because you already have a title after Table of Contents 👍 to make your readme file descriptive and professional 😊